corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -283,11 +283,11 @@
 
         {# always use angular js by default, as this will be our core library #}
         {% compress js %}
-        <script type="text/javascript" src="{% new_static 'style/lib/angular-1.4.4/angular.min.js' %}"></script>
-        <script type="text/javascript" src="{% new_static 'style/lib/angular-1.4.4/angular-route.min.js' %}"></script>
-        <script type="text/javascript" src="{% new_static 'style/lib/angular-1.4.4/angular-resource.min.js' %}"></script>
-        <script type="text/javascript" src="{% new_static 'style/lib/angular-1.4.4/angular-messages.min.js' %}"></script>
-        <script type="text/javascript" src="{% new_static 'style/lib/angular-1.4.4/angular-cookies.min.js' %}"></script>
+        <script type="text/javascript" src="{% new_static 'angular/angular.min.js' %}"></script>
+        <script type="text/javascript" src="{% new_static 'angular-route/angular-route.js' %}"></script>
```

Comment:
Nice, didn't realize we had a router. Another +1 for Bootstrap 3 pages -> a working back button!